code/obj/item.dm

```diff
@@ -307,11 +307,7 @@
 
 /obj/item/proc/set_health()
 	if (isnull(initial(src.health))) // if not overridden
```

Comment:
no need for this if anymore since you're checking the same thing in the proc

Reply:
It's checking for if there's a need to set the health

Reply:
right, but you're already doing the check in the proc?

Reply:
Yeah, I was meaning it felt odd to say src.health = initial(src.health) when it is already that value, but can change

Reply:
eh, probably doesn't matter either way